package.json

```diff
@@ -20,8 +20,7 @@
     "os-family": "^1.0.0",
     "pify": "^2.3.0",
     "pinkie": "^2.0.1",
-    "read-file-relative": "^1.2.0",
-    "viewport-list": "^3.0.1"
+    "read-file-relative": "^1.2.0"
```

Comment:
We need to add `data` dir to `"files"` in `package.json`. Otherwiser it will not be published